site/docs/components/file-drop-zone/usage.mdx

```diff
@@ -11,10 +11,22 @@ data:
 
 ### Using the component
 
-Make sure you communicate any limits, such as maximum file size and type restrictions, before the user uploads their file. If the upload fails, clearly explain the reason why and how to resolve it.
+#### Composition
+
+- When composing the file drop zone, consider using `FileDropZoneIcon` to inform users about the current status. You can show an upload icon when the user has not yet uploaded an item, or a status icon that communicates the current upload status.
+- We recommend passing a header to the component to indicate the expected action, such as "drop files here."
+- You can customize the `FileDropZoneTrigger`'s text by passing `children` into the component.
+- Communicate any requirements, such as maximum file size and type restrictions as validation text, before the user uploads their file. If the upload fails, clearly explain the reason why and how to resolve it.
+
+#### Validation and acceptance criteria
```

Comment:
"validation" is a concept only appearing in "how to" tab. I don't think we have anything in the examples tab associated with this. e.g. are we talking about text below the trigger button should be validation message?

Reply:
added some more validation context within the examples